src/dune_rules/melange/melange_rules.ml

```diff
@@ -241,6 +241,64 @@ let setup_emit_cmj_rules ~sctx ~dir ~scope ~expander ~dir_contents
   in
   Buildable_rules.with_lib_deps ctx compile_info ~dir ~f
 
+module Runtime_deps = struct
+  let to_action_builder ~expander dep_conf =
+    let runtime_deps =
+      let runtime_deps, _sandbox = Dep_conf_eval.unnamed ~expander dep_conf in
+      runtime_deps
+    in
+    Action_builder.memoize "melange runtime deps" runtime_deps
```

Comment:
Returning Memo.t doesn't mean the returned value is memoized. It just means that it might possibly be memoized. I agree though, I'm not sure I see the need to memoize the set of deps here. I don't see it being reused anywhere.